src/main/java/JsoupBadLinks.java

```diff
@@ -15,7 +15,12 @@
 
 public class JsoupBadLinks {
     private static final int BAD_CODE = 404;
-    private static final int BAD_CODE2 = 505;
+    private static final int BAD_CODE0 = 500;
+    private static final int BAD_CODE1 = 501;
```

Comment:
@dwaynewade0033 doYou need recognize types of 50x errors? Can You just check `code > 499 && code < 600`?